core/server/data/schema/schema.js

```diff
@@ -25,7 +25,13 @@ module.exports = {
         published_by: {type: 'string', maxlength: 24, nullable: true},
         custom_excerpt: {type: 'string', maxlength: 2000, nullable: true, validations: {isLength: {max: 300}}},
         codeinjection_head: {type: 'text', maxlength: 65535, nullable: true},
-        codeinjection_foot: {type: 'text', maxlength: 65535, nullable: true}
+        codeinjection_foot: {type: 'text', maxlength: 65535, nullable: true},
+        og_image: {type: 'string', maxlength: 2000, nullable: true},
+        og_title: {type: 'string', maxlength: 2000, nullable: true},
+        og_description: {type: 'string', maxlength: 2000, nullable: true},
+        twitter_image: {type: 'string', maxlength: 2000, nullable: true},
+        twitter_title: {type: 'string', maxlength: 2000, nullable: true},
+        twitter_description: {type: 'string', maxlength: 2000, nullable: true}
```

Comment:
We have to either reduce the length of the fields or transform them into type `text`, because the row limit exceeds. It's 65,535 bytes per table.